app/controllers/comments_controller.rb

```diff
@@ -1,6 +1,9 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
-  before_action :is_an_authorized_user, only: [:destroy, :create]
+  before_action :is_an_authorized_user, only: [:create]
```

Comment:
We can now delete this, we are replacing these logics with Pundit. It's a way to organize authorization.